src/main/java/seedu/rc4hdb/model/person/Fields.java

```diff
@@ -0,0 +1,18 @@
+package seedu.rc4hdb.model.person;
+
+import java.util.List;
+
+/**
+ *
+ */
+public class Fields {
```

Comment:
I like how you approached this and in fact, this has given me an idea of how to manage the Resident fields, though, it might be more convenient to declare each of the individual field identifiers, or whatever you call them in each of their respective classes, so for e.g, the Name class can have `NAME_FIELD` and so on. For now, you can leave it as it is.

Reply:
More info can be found in issue #77